### PR TITLE
Clickable each items on on-chain-data board

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Clickable each items on on-chain-data board

스크린샷 2023-06-16 오후 4 26 24

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

**Comment by sirius651** [comment]
after reviewing design QA, will write e2e test

File: src/components/dapp-staking/my-staking/OnChainData.vue [hunk 1]
```diff
@@ -153,6 +157,12 @@ export default defineComponent({
     );
     const isShiden = computed<boolean>(() => currentNetworkName.value === 'Shiden');
 
+    const goStakePageLink = (address: string | undefined): void => {
+      const base = networkParam + Path.DappStaking + Path.Stake;
```
Comment (impelcrypto) [inline src/components/dapp-staking/my-staking/OnChainData.vue:161]: Maybe go to dApp page instead of Staking page? @Kahonnohak

**Comment by Kahonnohak** [comment]
Cool!

- Could you change the status of the background only when hover please? So no button colour is seen when you first look at and appears when hover?

- Could you navigate to project pages rather than the staking page please?

**Comment by sirius651** [comment]
> Cool!
> 
> * Could you change the status of the background only when hover please? So no button colour is seen when you first look at and appears when hover?
> * Could you navigate to project pages rather than the staking page please?

Fixed in this [commit](https://github.com/AstarNetwork/astar-apps/pull/838/commits/10d6576151ce72859df923f52006e3ecb03cfd5d)

**Review by Kahonnohak — APPROVED** [review]
LGTM! Thank you!

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**